src/ParsePushAdapter.js

```diff
@@ -49,8 +51,19 @@ export class ParsePushAdapter {
     let deviceMap = classifyInstallations(installations, this.validPushTypes);
     let sendPromises = [];
     for (let pushType in deviceMap) {
-      let sender = this.senderMap[pushType];
+      var sender = this.senderMap[pushType];
       let devices = deviceMap[pushType];
+      
+      if (!sender) {
+        if (pushType == 'tvos') {
+          // if there is only an ios push configuration available reuse it for tvos
+          sender = this.senderMap['ios'];
+        } else if (pushType == 'ios') {
+          // if there is only a tvos push configuration available reuse it for ios
+          sender = this.senderMap['tvos'];
```

Comment:
I personally think is is more transparent if we do not add this redundancy. If someone needs support for multiple push targets he needs to define all of them in his config. He can use the same certificate of course.